bot/exts/utils/snekbox.py

```diff
@@ -480,6 +491,13 @@ async def run_job(
                 break
             log.info(f"Re-evaluating code from message {ctx.message.id}:\n{code}")
 
+    async def eval_ctx_menu(self, interaction: Interaction, message: Message) -> None:
+        """Same functionality as `eval_command` but invoked using a context menu."""
+        fake_ctx: Context = await Context.from_interaction(interaction)
+        python_version = "3.11"
```

Comment:
Don't we keep track of the python version that we usually use for evaluation ? 
Having this here can lead to us forgetting about it once we update the py version